Lib/difflib.py

```diff
@@ -1111,6 +1112,10 @@ def unified_diff(a, b, fromfile='', tofile='', fromfiledate='',
     For inputs that do not have trailing newlines, set the lineterm
     argument to "" so that the output will be uniformly newline free.
 
+   Set *color* to ``True`` to enable output in color, similar to
+   :program:`git diff --color`. Even if enabled, it can be
+   :ref:`controlled using environment variables <using-on-controlling-color>`.
```

Comment:
The CPython docstrings aren't rendered into docs, so let's use plain formatting like elsewhere. Perhaps:

```suggestion
   Set 'color' to True to enable output in color, similar to
   'git diff --color'. Even if enabled, it can be
   controlled using environment variables such as 'NO_COLOR'.
```

Reply:
Done. Also fixed the 3 spaces thing :facepalm: